controllers/categoriesController.js

```diff
@@ -0,0 +1,33 @@
+const BaseController = require("./baseController");
+
+class CategoriesController extends BaseController {
+  constructor(model) {
+    super(model);
+  }
+
+  async getCategory(req, res) {
```

Comment:
```suggestion
  async getCategories(req, res) {
```

plural as it returns potentially all categories